src/pages/Search/Search.js

```diff
@@ -1,6 +1,178 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import PopularSearch from '../../components/PopularSearch/PopularSearch';
+import EmptySearchResult from '../../components/EmptySearchResult/EmptySearchResult';
+import SearchResult from '../../components/SearchResult/SearchResult';
 import styled from 'styled-components';
+import { flexSet, commonLayOut } from '../../styles/mixin';
+import POPULAR_ARR from '../../components/PopularSearch/PopularSearchData';
 
-const Search = () => {};
+const Search = () => {
+  const [searchValue, setSearchValue] = useState('');
+  const [isButtonOn, setIsButtonOn] = useState(false);
+  const [searchResultArr, setSearchResultArr] = useState([]);
+  const [popularSearchTermsOn, setPopularSearchTermsOn] = useState(true);
+  const history = useHistory();
+
+  const handleSearchInput = event => {
+    const {
+      target: { value },
+    } = event;
```

Comment:
준우님이랑 같네요! ㅋㅋ 
쓰려고 하는 변수만 잘 보이게 해주세요!
```suggestion
    const { value } = event.target;
```

Reply:
ㅋㅋㅋ 넵 수정했습니다!